Website/components/AboutPage.tsx

```diff
@@ -0,0 +1,343 @@
+  return `https://api.github.com/repos/SaptarshiSarkar12/Drifty/contributors?per_page=100&page=${
+    pageIndex + 1
+  }`;
+};
+
+export default function AboutPage() {
+  const { data: repoData, error: repoError } = useSWR<RepoData>(
+    "https://api.github.com/repos/SaptarshiSarkar12/Drifty",
+    fetcher
+  );
+
+  const {
+    data: pages,
+    error: contribError,
+    isLoading,
+    size,
+    setSize,
+  } = useSWRInfinite<Contributor[]>(getKey, fetcher, {
+    dedupingInterval: 60000,
+    revalidateOnFocus: false,
+  });
+
+  // Automatically load next page if there might be more data
+  useEffect(() => {
+    if (pages && pages[pages.length - 1]?.length === 100) {
+      setSize(size + 1);
+    }
+  }, [pages, setSize, size]);
+
+  // Flatten all pages into single array
+  const allContributors = pages ? pages.flat() : [];
+
+  // Filter and process contributors
+  const humans = allContributors.filter((c) => c.type === "User");
+  const bots = allContributors.filter((c) => c.type === "Bot");
+  const maxVisible = 14;
+  const visibleContributors = humans.slice(0, maxVisible);
+  const otherContributorsCount =
+    Math.max(humans.length - maxVisible, 0) + bots.length;
+
+  // Error boundaries
+  if (repoError)
+    return (
+      <div className="min-h-screen flex items-center justify-center">
+        Failed to load repository data
+      </div>
+    );
+  if (contribError)
+    return (
+      <div className="min-h-screen flex items-center justify-center">
+        Failed to load contributors
+      </div>
+    );
+
+  return (
+    <div className="min-h-screen p-6 sm:p-8 ">
+      <div className="max-w-6xl mx-auto space-y-12">
+        {/* Header */}
+        <header className="text-center space-y-6">
+          <h1 className="text-5xl font-bold ">Drifty</h1>
+          <p className="text-xl max-w-3xl mx-auto leading-relaxed">
+            An open-source, multi-platform download utility designed with
+            simplicity and efficiency in mind.
+          </p>
+        </header>
+
+        {/* Stats Grid */}
+        <div className="grid md:grid-cols-3 gap-6">
+          <div className="p-6 rounded-xl shadow-md">
+            <h3 className="text-sm font-medium mb-2">License</h3>
+            <div className="flex items-center space-x-2">
+              <div className="h-8 w-8 bg-green-100 dark:bg-green-900 rounded-lg flex items-center justify-center">
+                <GrLicense className="text-green-600 dark:text-green-300" />
+              </div>
+              <a
+                href="https://github.com/SaptarshiSarkar12/Drifty/blob/master/LICENSE"
+                className="text-lg font-medium hover:underline text-gray-900 dark:text-white"
+                aria-label="View Apache 2.0 License"
+              >
+                Apache 2.0
+              </a>
+            </div>
+          </div>
+
+          <div className="p-6 rounded-xl shadow-md">
+            <h3 className="text-sm font-medium mb-2 ">Technology</h3>
```

Comment:
@abhilashreddysh Can you also add links to their (all three mentioned under `Technology`) respective websites just like the Website tech links?
Java: https://www.java.com/en/
GraalVM: https://graalvm.org/
GluonFx: https://github.com/gluonhq/gluonfx-maven-plugin/